### DISCUSSION TITLE
Can someone release an official update already?

### DISCUSSION THREAD

**Discussion by vincent-savysoda** [Facepunch/Facepunch.Steamworks#688] [General]
Can we please roll out a new official update already? It's been two years now since the last one and some important major changes have been merged but has not been released yet. Any info on when this will happen?

**Comment by muzboz** [comment]
I'm not totally sure, but I thought this was kind of a project shared by a
studio who made this for themselves, and shared it to others to be nice?

(I may be wrong about that!)

But if I'm right, then I guess we should all encourage an official update
with words of gratefulness and applause, if we want to get them inspired
and motivated to roll out an official update.

Seems like a cool resource.  So I hope to see it continue, as I think I'll
be trying to use it some time soon!

Just not sure we should be poking and prodding them, as if they're a
service provider who owe users for some paid product or something...?

It's just a project they share out of the kindness of their hearts isn't it?

(I actually don't quite know the full nature of the project, so excuse me
if I've got my wires crossed).

- Murray

**Reply by vincent-savysoda** [reply]
Sorry I didn't mean to sound aggressive or anything. Was just abit frustrated as I've been waiting on an update for a long time now. I know this is a free resource but I would at least hope to get some information about whether or not an update will be coming so we can make a decision whether to move on to a different solution. It's been more than 2 years already.

**Reply by vincent-savysoda** [reply]
It's just doubly annoying because the fix for some showstopper bugs(alongside the major update to the SDK) was merged like a month ago and I had held off on switching to Steamworks.NET(since that was being updated regularly despite apparently being more annoying to implement) as I was hoping an official update will be rolled out soon. I also tagged Garry in another thread to ask for a potential release date but no response.

**Comment by muzboz** [comment]
Cheers for the update there, from a user's perspective.

I'm sure I'll probably find myself in a tricky position too when it comes
time to implement the Steam API into my Unity game, as I'd love to have a
nice working solution that makes it easy!

And yes, it does seem like there's probably not currently a clear choice as
to whether to use Facepunch Steamworks, or something else.

I've subscribed to these emails to keep tabs on seeing what state it's in,
ahead of having to choose a solution!  So it's useful to see people's
frustration to be honest, as it helps me to know where this particular
solution is at.

*I won't try to propose some sort of solution going forwards, as I'm out of
my element entirely!  *

*But I wonder if there would be a way for Facepunch to perhaps lay out a
bit of a Roadmap, or a general indication of their views and intentions
around Facepunch Steamworks in coming months, or possibly whether there's
anything the community around it can do to help streamline or lubricate the
process going forwards?*

Again, I'm sort of suggesting things that are beyond my actual
understanding, as I'm not actually using the resource currently!  :)

But I'm aiming to use it in coming months, so if there's anything the users
can do out here, let us know Facepunch.  :D

All the best all.
